backend/btrixcloud/crawls.py

```diff
@@ -1034,7 +1034,16 @@ async def download_qa_run_as_single_wacz(
         if not qa_run.resources:
             raise HTTPException(status_code=400, detail="qa_run_no_resources")
 
-        resp = await self.storage_ops.download_streaming_wacz(org, qa_run.resources)
+        metadata = {
+            "type": "qa_run",
+            "id": qa_run_id,
+            "crawlId": crawl_id,
```

Comment:
I might suggest we stick to one of camelcase or snake case here instead of mixing (e.g. `qa_run` and `crawlId`)

Reply:
will go with `qaRun`